test/Makefile

```diff
@@ -9,10 +9,16 @@ GTEST_SRCS_=$(GTEST_SOURCEDIR)/*.cc $(GTEST_SOURCEDIR)/*.h $(GTEST_HEADERS)
 # TODO: Factor out the common variables.
 CPP_NETLIB_DIR=$(LIBDIR)/cpp-netlib
 YAML_CPP_DIR=$(LIBDIR)/yaml-cpp
+YAML_CPP_LIBDIR=$(YAML_CPP_DIR)
+SUBMODULE_DIRS=$(CPP_NETLIB_DIR) $(YAML_CPP_DIR)
```

Comment:
Is this used anywhere? If not, let's remove it.

Reply:
Removed.